examples/default.py

```diff
@@ -11,13 +12,14 @@ class Foo:
     s: str = 'foo'
     f: float = field(default=100.0)  # Use dataclass field.
     b: bool = field(default=True)
+    d: Dict[str,int] =field(default_factory=dict)
 
 
 def main():
     h = Foo()
     print(f"Into Json: {to_json(h)}")
 
-    s = '{"i": 10, "s": "foo", "f": 100.0, "b": true}'
+    s = '{"i": 10, "s": "foo", "f": 100.0, "b": true, "d":{}}'
```

Comment:
Oops, sorry. 

Your code looks good to me, but can you please try to fix this line as follows?

```python
s = '{}'
```

you should still see the output like this.
```
From Json: Foo(i=10, s='foo', f=100.0, b=True, d={})
```

Because this is an example of `default` and `default_factory` field, `s = '{}'` makes `Foo` class uses the default/default_factory during deserialization. So I thought it's more appropriate 🙂

Reply:
Ok . I will do it

Reply:
LGTM! Thank you! 👍